src/errors.js

```diff
@@ -57,15 +57,31 @@ export function onlyInstancesOf(
  *      // error.code is guaranteed to be ENOENT
  *    }))
  *
+ *  or:
+ *
+ *  Promise.resolve()
+ *    .catch(onlyErrorsWithCode(['ENOENT', 'ENOTDIR'], (error) => {
+ *      // ...
+ *    }))
+ *
  * All other errors will be re-thrown.
  *
  */
 export function onlyErrorsWithCode(
-    codeWanted: string, errorHandler: Function): Function {
+    codeWanted: string | Array<string>, errorHandler: Function): Function {
   return (error) => {
-    if (error.code !== codeWanted) {
+    let throwError = true;
+
+    if (codeWanted instanceof Array && codeWanted.indexOf(error.code) !== -1) {
```

Comment:
This can be ```Array.isArray(codeWanted)``` (but it is not very important because this code runs in node, and so it will not make a great difference)